docs/roles/system_manager/manage_users.md

```diff
@@ -64,6 +64,40 @@ Once you're certain that you're adding a new user, make sure that the following
 - Be particularly careful never to use them to log in to any user-accessible VMs (such as the SRDs)
 ```
 
+### {{woman}} {{man}} Adding users to security group(s)
```

Comment:
Should we also explain removing users from groups (or modifying groups)?

Reply:
Perhaps "Managing security group(s)" which can include addition and deletion

Reply:
:+1:

If it starts to get very large it might be worth breaking it down into addition, deletion, _etc._